docs/reference/mapping/types/boolean.asciidoc

```diff
@@ -241,8 +241,7 @@ any issues, but features in technical preview are not subject to the support SLA
 of official GA features.
 
 `boolean` fields support <<synthetic-source,synthetic `_source`>> in their
-default configuration. Synthetic `_source` cannot be used together with
-<<copy-to,`copy_to`>> or with <<doc-values,`doc_values`>> disabled.
+default configuration.
 
 Synthetic source always sorts `boolean` fields. For example:
```

Comment:
Another note for a follow up is to replace "always sorts" with "may sort" or "does not preserve order" in cases like this.

Reply:
Maybe we can have a single page that captures all this.

Reply:
I will just say "may" change because usually that statement is followed by an example where the order actually is different. So I think the sentence is relevant to understand the example.